src/main/java/net/masterthought/cucumber/generators/TagsOverviewPage.java

```diff
@@ -38,14 +36,14 @@ public void prepareReport() {
         context.put("chart_data", generateTagValues(filterExcludedTags(tags)));
     }
 
-    static String generateTagLabels(List<TagObject> tagsObjectList) {
+    static String[] generateTagLabels(List<TagObject> tagsObjectList) {
         int tagCount = tagsObjectList.size();
         String[] tagNames = new String[tagCount];
 
         for (int i = 0; i < tagCount; i++) {
-            tagNames[i] = StringUtils.wrap(tagsObjectList.get(i).getName(), "\"");
+            tagNames[i] = tagsObjectList.get(i).getName();
```

Comment:
Previously the function had two responsibilities. 1) transform the the list of objects into an array of strings and 2) translate that array of strings into a string that represents a javascript array of strings. The latter was also done in two different places in two different ways.  This has been split into two functions and the redundancy has been removed.

Both the transformation and the translation are now tested from end to end which wasn't done before as the tests did not look at the generated javascript. So in all I reckon we're up from where we were.